checkov/common/runners/runner_registry.py

```diff
@@ -283,3 +285,13 @@ def get_enriched_resources(repo_roots: List[Union[str, os.PathLike]]) -> Dict[st
                                 "skipped_checks": skipped_checks,
                             }
         return enriched_resources
+
+    @staticmethod
+    def strip_code_blocks_from_json(report_jsons):
```

Comment:
```suggestion
    def strip_code_blocks_from_json(report_jsons: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
```
this method is actually non-idempotent, therefore it doesn't need to return the `report_jsons` they will be changed in-place, because `dict` in Python is a mutable object.
If you decide to remove the return statement then the type hints will be
```python
def strip_code_blocks_from_json(report_jsons: List[Dict[str, Any]]) -> None:
```

Reply:
Thanks for clarifying this point